srcpkgs/evolution-data-server/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'evolution-data-server'
 pkgname=evolution-data-server
-version=3.28.4
-revision=3
+version=3.30.0
+revision=1
 build_style=cmake
 configure_args="-DENABLE_GOOGLE_AUTH=OFF -DENABLE_UOA=OFF
```

Comment:
Please remove -DENABLE_GOOGLE_AUTH as it is an unused option, see https://github.com/void-linux/void-packages/pull/2400